lib/models/favorites_models/favorite.dart

```diff
@@ -44,7 +44,7 @@ class Data {
   Data.fromJson(Map<String, dynamic> json) {
     currentPage = json['current_page'];
     if (json['data'] != null) {
-      productData = new List<ProductData>();
+      productData =  [];
```

Comment:
Tell me The reason of change this file and how affect this branch?

Reply:
it's not affect this branch it came across of me and I edit it